paper/basic_training.tex

```diff
@@ -310,6 +310,10 @@ \subsubsection{Key concepts}
 In this type of non-equilibrium situation, the distribution of configurations will not follow a Boltzmann factor distribution.  
 Specialized simulation approaches are available to study such systems~\cite{Chong:2017:CurrentOpinioninStructuralBiology,  Zuckerman:2017:AnnuRevBiophys} but they are not beginner-friendly.  
 Non-equilibrium molecular concepts pertinent to cell biology have been discussed at an introductory level (e.g. \url{http://www.physicallensonthecell.org/}).
+Notably, many experiments are conducted at non-equilibrium conditions; for example, membrane diffusion coefficients are commonly measured by setting up a concentration gradient across the membrane and measuring the flux.
+It can be tempting to the beginner to setup an MD simulation in the same manner as such an experiment.
+However, maintaining non-equilibrium conditions is typically more complicated in an MD simulation than in an experiment as large reservoirs are commonly required.
+Frequently there exist equilibrium methods that can be used to obtain the same or similar information as a non-equilibrium experiment; users should seek to obtain familiarity with such methods before choosing to conduct a non-equilibrium MD simulation.
```

Comment:
I tend to not like the "there exist" phrasing; it's more direct to just be direct, e.g. "Frequently, equilibrium methods can provide ..."

Reply:
(I can fix this after merging this.)